.github/workflows/ci-cd.yml

```
@@ -1,182 +1,180 @@
+        with:
+          directory: .
+
+      - name: Prepare image variables
+        id: image_vars
+        run: |
+          echo "IMAGE_REPO=${{ env.AWS_ACCOUNT_ID }}.dkr.ecr.${{ env.AWS_REGION }}.amazonaws.com/${{ env.ECR_REPOSITORY }}" >> $GITHUB_ENV
+          echo "IMAGE_TAG=${GITHUB_SHA::7}" >> $GITHUB_ENV
+          echo "IMAGE_URI=${IMAGE_REPO}:${IMAGE_TAG}" >> $GITHUB_ENV
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.12)

[error] the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)